### PR TITLE
feat: CCIE-2069: Add liveness and readiness timeouts

### PR DESCRIPTION
**Pull request by dspadea**
<details open>
  <summary><a href="https://czi-tech.atlassian.net/browse/CCIE-2069" title="CCIE-2069" target="_blank">CCIE-2069</a></summary>
  <br />
  <table>
    <tr>
      <th>Summary</th>
  <td>Support liveness and readiness probe timeouts in happy-service-eks</td>
    </tr>
    <tr>
      <th>Type</th>
      <td>
        Task
        Task
      </td>
    </tr>
    <tr>
      <th>Status</th>
      <td>In Progress</td>
    </tr>
  </table>
</details>

---

Add timeout parameters for liveness and readiness probes

### CODE REVIEW

File: terraform/modules/happy-service-eks/main.tf [hunk 1]
```diff
@@ -313,6 +315,7 @@ resource "kubernetes_deployment_v1" "deployment" {
 
               initial_delay_seconds = container.value.initial_delay_seconds
               period_seconds        = container.value.period_seconds
+              timeout_seconds       = container.value.liveness_timeout_seconds
```
Comment (hspitzley-czi) [inline terraform/modules/happy-service-eks/main.tf:318]: To do this you'll also need to add the new field to the `sidecars` variable

File: terraform/modules/happy-service-eks/main.tf [hunk 2]
```diff
@@ -324,6 +327,7 @@ resource "kubernetes_deployment_v1" "deployment" {
 
               initial_delay_seconds = container.value.initial_delay_seconds
               period_seconds        = container.value.period_seconds
+              timeout_seconds       = container.value.readiness_timeout_seconds
```
Comment (hspitzley-czi) [inline terraform/modules/happy-service-eks/main.tf:330]: To do this you'll also need to add the new field to the `sidecars` variable

File: terraform/modules/happy-service-eks/variables.tf [hunk 3]
```diff
@@ -137,6 +137,18 @@ variable "period_seconds" {
...
+  type        = number
+  default     = 30
+  description = "Timeout for liveness probe."
+}
+
+variable "readiness_timeout_seconds" {
+  type        = number
+  default     = 30
+  description = "Readiness probe timeout seconds"
+}
```
Comment (hspitzley-czi) [inline terraform/modules/happy-service-eks/variables.tf:150]: I think you'll also want to add these to happy-stack-eks since that's the top-level module

Comment (dspadea) [reply terraform/modules/happy-service-eks/variables.tf:150]: Done! Added to stack variables and main.tf service definition.

**Review by jakeyheath — APPROVED** [review]
TY DAVE

**Pull request merged**